web/src/cypress/integration/users.ts

```diff
@@ -50,6 +50,20 @@ function testUsers(screen: ScreenFormat): void {
       })
     })
   })
+
+  describe('Page Actions', () => {
+    it('should edit a user role', () => {
+      cy.adminLogin()
+
+      cy.fixture('users').then((users) => {
+        cy.visit(`/users/${users[0].id}`)
+
+        cy.pageAction('Edit')
+        cy.get('[type="checkbox"]').check()
+        cy.dialogFinish('Confirm')
```

Comment:
Since we don't have any indication on the user page if someone is an admin or not, can we open the dialog again and validate that the checkbox is still filled as a sanity check?